src/plugins/bubble/bubble.js

```diff
@@ -0,0 +1,167 @@
+					} );
+				} else {
+					$element.css( {
+						bottom: bottomY
+					} );
+				}
+			};
+
+			// Correct bubble positionning on load, on resize an on Y scroll if necessary
+			stickyUntilFooter( $selector );
+
+			$( window ).on( "resize scroll", function() {
+				stickyUntilFooter( $selector );
+			} );
+		} else {
+			$( "main" ).after( "<footer id=\"wb-info\" class=\"" + componentName + "-mrgn" + "\"></footer>" );
+		}
+	},
+
+
+	setNotificationStatusWithExpiry = function( key, ttl ) {
+		var now = new Date();
+		localStorage.setItem( key, now.getTime() + ttl );
+	},
+
+	getNotificationStatusWithExpiry = function( key ) {
+		var expiryTimeValue = localStorage.getItem( key );
+		var now = new Date();
+
+		// compare the expiry time of the item with the current time
+		if ( now.getTime() > expiryTimeValue ) {
+
+			// If the item is expired, delete the item from storage
+			// and return null
+			localStorage.removeItem( key );
+			return null;
+		}
+		return expiryTimeValue;
+	},
+
+
+/**
+	 * @method init
+	 * @param {jQuery Event} event Event that triggered the function call
+	 */
+	init = function( event ) {
+
+		// Start initialization
+		// returns DOM object = proceed with init
+		// returns undefined = do not proceed with init (e.g., already initialized)
+		var elm = wb.init( event, componentName, selector ),
+			$elm;
+		if ( elm ) {
+			$elm = $( elm );
+
+			// Initiate dictionary
+			i18nDict = i18nDict[ wb.lang ] || i18nDict.en;
+
+			var $bubbleElm = $( "<div class='" + componentName + "-wrap'></div>" );
+			$elm.wrap( $bubbleElm );
+
+			// Get notification status to check if it is still valid after TTL
+			isNotif = getNotificationStatusWithExpiry( componentName + "-notif" );
+
+			var $notification = ( !isNotif ? "<p class='trans-left'>" +
+			"<span class='notif'>" + $elm.text() + "</span>" +
+			"<button class='notif-close' title='" + i18nDict.closeNotif + "' aria-label='" + i18nDict.closeNotif + "'>&times;</button></p>" : "" );
+
+			$elm.parent().append( $notification );
+
+			// Add linke to the top page skip link.
+			var list = document.getElementById( "wb-tphp" );
+			if ( list ) {
+
+				var li = document.createElement( "li" );
+				li.className = "wb-slc";
+
+				// Append the Basic HTML version link version
+				if ( $elm.attr( "data-wb-doaction" ) ) {
+					li.innerHTML = "<button  data-wb-doaction='" + $elm.attr( "data-wb-doaction" ) + "' class=\"wb-sl\" >" + i18nDict.skip + " " + $elm.text() + "</button>";
+				}
+
+				// Add button at the 2nd poisiton of the li
+				list.insertBefore( li, list.childNodes[ 2 ] );
+			}
+
+			// Initiate chat wizard bubble
+			adjustBubbleMargin( $elm.parent() );
+
+			// Identify that initialization has completed
+			wb.ready( $elm, componentName );
+		};
+	};
+
+// Close notification aside bubble
+$document.on( "click",  selector + "-wrap .notif-close", function( event ) {
```

Comment:
The bubble click should actually do the same thing as the notification close, meaning it:
1. closes the notification;
2. sets a local storage; and
3. removes the pulsing animation.

```suggestion
$document.on( "click",  selector + "-wrap .notif-close, " + selector, function( event ) {
```